manim/scene/scene.py

```diff
@@ -876,8 +876,28 @@ def add_sound(self, sound_file, time_offset=0, gain=None, **kwargs):
             The offset in the sound file after which
             the sound can be played.
 
-        gain :
+        gain : float
+            Amplification of the sound.
 
+        Examples
+        --------
+        .. manim:: SoundExample
+
+            class SoundExample(Scene):
+                # Source of sound under Creative Commons 0 License. https://freesound.org/people/Druminfected/sounds/250551/
+                def construct(self):
+                    dot = Dot().set_color(GREEN)
+                    self.add_sound("click.wav")
+                    self.add(dot)
+                    self.wait()
+                    self.add_sound("click.wav")
+                    dot.set_color(BLUE)
+                    self.wait()
+                    self.add_sound("click.wav")
+                    dot.set_color(RED)
+                    self.wait()
+
+        Download the resource for the following example `here <https://github.com/ManimCommunity/manim/blob/master/docs/source/_static/click.wav>`_ .
```

Comment:
Either change "following example" to "previous example", or move the text above the example.

Reply:
done! :)